source/resource/visu_config.xsd

```diff
@@ -773,11 +773,16 @@
     </xsd:sequence>
     <xsd:attribute name="name" type="xsd:string" use="optional">
     	<xsd:annotation>
-            <xsd:documentation xml:lang="en">Name/title of the group</xsd:documentation>
-            <xsd:documentation xml:lang="de">Name/Überschrift der Gruppe</xsd:documentation>
+          <xsd:documentation xml:lang="en">Name/title of the group</xsd:documentation>
+          <xsd:documentation xml:lang="de">Name/Überschrift der Gruppe</xsd:documentation>
+        </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="nowidget" type="xsd:boolean" use="optional">
+      <xsd:annotation>
+          <xsd:documentation xml:lang="en">Removes the frame borders inside the group</xsd:documentation>
+          <xsd:documentation xml:lang="de">Unterdrückt die Rahmen innerhalb der Gruppe</xsd:documentation>
```

Comment:
Eigentlich gilt diese Aussage nur für das Metal-Design, sollte als Hinweis darein. In den anderen Designs hat nowidget meines Wissens keine Auswirkungen.